chrome_extension/record.py

```diff
@@ -1,14 +1,46 @@
 import asyncio
 import websockets
+import sqlite3
+import json
 
-async def print_message(message):
+# Function to initialize the database
+def init_db():
+    conn = sqlite3.connect('chrome.db')
+    c = conn.cursor()
+    c.execute('''
+        CREATE TABLE IF NOT EXISTS browser_event (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            timestamp DATETIME DEFAULT CURRENT_TIMESTAMP,
+            message TEXT
+        )
+    ''')
+    conn.commit()
+    conn.close()
+
+# Function to insert a message into the database
+def insert_message(message):
+    conn = sqlite3.connect('chrome.db')
+    c = conn.cursor()
+    c.execute('''
+        INSERT INTO browser_event (message)
+        VALUES (?)
+    ''', (message,))
+    conn.commit()
+    conn.close()
```

Comment:
What do you think about adding a method to `crud.py` for this?

Reply:
I do not think so, I will make use of  the ones that we alredy have, This was jsut for prototype. Now, the next step will be to to integrate this DOMs into the openadapt.db file in the same manner as windowd events are done.

Reply:
I hope this answers the comments  :)